src/main/java/io/zeebe/monitor/config/JacksonConfig.java

```diff
@@ -0,0 +1,19 @@
+package io.zeebe.monitor.config;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.zeebe.protocol.jackson.ZeebeProtocolModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class JacksonConfig {
+
+  @Bean
+  public ObjectMapper objectMapper() {
+    var objectMapper = new ObjectMapper();
+    objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    objectMapper.registerModule(new ZeebeProtocolModule());
```

Comment:
Clarification question: what is this used for?
Guessing, Kafka does not use protobuf binary messages, but JSON instead?

Reply:
Yes, it is a JSON serialization library. Our exporter in this feature is https://github.com/camunda-community-hub/zeebe-kafka-exporter, which uses it to serialize kafka messages. So I need it to deserialize messages.